HISTORY.md

```diff
@@ -8,6 +8,7 @@ unreleased
   * deps: path-to-regexp@3.0.0
   * deps: eslint@6.8.0
   * deps: eslint-plugin-markdown@1.0.1
+  * Store matched routes in request #34 #86
```

Comment:
Typically we do not put a hash to numbers in here that refer to github numbers. Just leave it as the short change description.

Reply:
I added that because this was more than one PR, but I am not opposed to it being removed.  @gabegorelick if you make more changes and want to fix this feel free.  Otherwise we can wait until later.

Reply:
The github numbers are just included in the body of the git commit message itself, mainly because that is where github will acknowledge them by making them links and closing the prs on merge automatically .

Reply:
Updated. I also added references to #34 and #85 (the issue that this will fix) in the commit message.